README.md

```diff
@@ -85,8 +85,8 @@ Hard coding instance URL is not a best practice as they reduce the usability of
 ### Before Business rules should not insert() records in any tables
 Before business rules execute before the data on current record is saved to database. In case, the before business rule fails or aborts its operation, then any insert operations within the business rule becomes invalid but it cannot be reverted back as the operation would have already been completed. In order to avoid such issues, before business rules should not include insert operations.
 
-### Update set description should not be empty
-Validates the description of the update sets created is not empty as it provides the release management team better understanding what's getting pushed.
+### Catalog UI policy should be used in either a Catalog Item or a Variable Set
+Validates that Catalog UI Policies are either associated with a Catalog Item or a Variable Set. Catalog UI Policies that are not in use should be deleted as they are unnecessary.
```

Comment:
This is not the entry that needed to be updated. This should be left as is, the title that is in error is down around line 103